src/Controls/tests/TestCases.HostApp/Issues/Issue27614.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 27614, "Label not sized correctly on Android", PlatformAffected.Android)]
+	public class Issue27614 : ContentPage
+	{
+		public Issue27614()
+		{
+			var label = new Label
```

Comment:
Not required, and already tested locally. But could be nice if the test could have 2 Labels: one with a single line like "Hello World" and other with several lines.